<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewTable</class>
 <widget class="QDialog" name="DialogNewTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>911</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>621</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>表单名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditName">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>成交日期</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>进货</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>出货</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
